### PR TITLE
Change NER to use metrics class

### PR DESCRIPTION
**Pull request by neubig**
Fixes https://github.com/neulab/ExplainaBoard/issues/190

### CODE REVIEW

File: explainaboard/utils/span_utils.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+
+def get_spans_from_bio(seq):
```
Comment (pfliu-nlp) [inline explainaboard/utils/span_utils.py:4]: I actually haven't figured out the content of `seq`  especially after reading the docstring, should it be list[str], such as ["O", "B-PER", "I-PER"]?

**Review by pfliu-nlp — APPROVED** [review]
Overall, it looks nice to me.

File: explainaboard/utils/span_utils.py (hunk 1 above)
Comment (neubig) [reply explainaboard/utils/span_utils.py]: Thanks! This doc was legacy doc from before, but I made it clearer (and also added a test checking the desired behavior)

Comment (pfliu-nlp) [reply explainaboard/utils/span_utils.py]: nice! Feel free to merge this PR!

**Comment by neubig** [comment]
Thanks for the review! I also bumped the version and fixed the EaaS interface to use the latest version of the client. This will be release 0.8.5

**Pull request merged**